dsl.md

```diff
@@ -91,21 +94,56 @@ Additionally, workflows may optionally accept inputs and produce outputs, allowi
 
 #### Status Phases
 
-Workflows in the Serverless Workflow DSL can exist in several phases, each indicating the current state of the workflow execution. These phases include:
+Both workflows and tasks in the Serverless Workflow DSL can exist in several phases, each indicating the current state of the workflow/task execution. These phases include:
 
 | Phase |	Description |
 | --- | --- |
-| `pending` |	The workflow has been initiated and is pending execution. |
-| `running` |	The workflow is currently in progress. |
-| `waiting` |	The workflow execution has been paused or halted temporarily and is waiting for something to happen. |
-| `cancelled` |	The workflow execution has been terminated before completion. |
-| `faulted` |	The workflow execution has encountered an error. |
-| `completed` |	The workflow execution has successfully finished all tasks. |
+| `pending` |	The workflow/task has been initiated and is pending execution. |
+| `running` |	The workflow/task is currently in progress. |
+| `waiting` |	The workflow/task execution is temporarily paused, awaiting either inbound event(s) or a specified time interval as defined by a [`wait`](https://github.com/serverlessworkflow/specification/blob/main/dsl-reference.md#wait) task. |
+| `suspended` |	The workflow/task execution has been manually paused by a user and will remain halted until explicitly resumed. |
+| `cancelled` |	The workflow/task execution has been terminated before completion. |
```

Comment:
```suggestion
| `cancelled` |	The workflow/task execution has been terminated before completion. |
```

> While both canceled and cancelled are acceptable for the past tense of cancel, the version with one L is more common in American English, and the version with two L's is more common in British English.

🤷

Reply:
Maybe we should use the single L?

Reply:
Well, UK vs USA. I have no particular preference, so yeah, as you wish 😉

Reply:
On a side note, maybe we should add that we should always prefer US english vs UK one in [here](https://github.com/serverlessworkflow/specification/blob/main/dsl.md#design).

Reply:
> Well, UK vs USA. I have no particular preference, so yeah, as you wish 😉

Which is the accepted form in Irish English? ;)

Reply:
Answering myself https://prowritingaid.com/cancelled-vs-canceled. 
Also, English is named english because is originally from England, so, in case of doubt, lets always use the original.  I vote for double L.